include/seqan3/io/stream/debug_stream.hpp

```diff
@@ -329,6 +329,23 @@ inline debug_stream_type & operator<<(debug_stream_type & s, variant_type && v)
     return s;
 }
 
+/*!\brief A std::optional can be printed by printing it's value or nothing if valueless.
+ * \tparam    optional_type The underlying type of the variant.
+ * \param[in] s             The seqan3::debug_stream.
+ * \param[in] arg           The std::optional value.
```

Comment:
```suggestion
 * \param[in] arg           The std::optional.
```

If you write `value` I would think the `std::optional::value()` value